spec/support/carrierwave.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.configure do |config|
+  config.after(:all) do
+    FileUtils.rm_rf(Dir[Rails.root.join('spec', 'fixtures', 'uploads')])
```

Comment:
Is it likely that this might cause some test flicker if we only clean up after the whole test suite is run? Could we tag any carrierwave tests and cleanup after each one, to ensure we've got a clean working directory?

Reply:
As we use this root directory further down too, it might be worth having it around as a variable too

Reply:
Thank you for the great spots!

I've refactored the tests to delete after each relevant test as completed, and DRY'd up how we store the fixtures path.